Proiect/src/Cinema/Cinema.java

```diff
@@ -0,0 +1,60 @@
+package Cinema;
+
+import java.util.ArrayList;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Set;
+
+public class Cinema {
+    private Adresa adresa;
+    private Set<String> filme;
+    Hashtable<String, Integer> NrOreDeschisPeZi = new Hashtable<>(7);
```

Comment:
[Fix] acest camp trebuie sa fie privat, iar numele sa fie cu camelCase, sa nu inceapa cu litera mica. De asemenea recomand HashMap in loc de Hashtable, nu cred ca am vazut niciodata aceasta clasa sa fie folosita